props-core/src/main/java/sh/props/SubscribableProp.java

```diff
@@ -133,36 +135,36 @@ protected void onValueUpdate(@Nullable T value, long epoch) {
       return;
     }
 
-    ForkJoinPool.commonPool()
-        .execute(
-            () -> {
-              long current =
-                  this.lastProcessedEpoch.updateAndGet(SubscribableProp.setIfMoreRecent(epoch));
-
-              // the epoch was not updated, which means that the current update event should be
-              // discarded
-              if (current != epoch) {
-                return;
-              }
-
-              // lock to ensure that the updated value was received by all consumers
-              // before allowing a new update to be processed
-              this.sendStage.lock();
-
-              try {
-                // since some time may have passed, recheck that this epoch is still valid
-                if (this.lastProcessedEpoch.get() != epoch) {
-                  // if not, discard the update
-                  return;
-                }
-
-                // send the same value to all consumers
-                this.updateHandlers.forEach(c -> c.accept(value));
-              } finally {
-                // allow the next update to be sent
-                this.sendStage.unlock();
-              }
-            });
+    Scheduler runner = scheduler != null ? scheduler : Scheduler.instance();
```

Comment:
Allows offloading async processing to a custom scheduler (provided by the Registry, upon `bind`ing)